washingtonpost/elex-live-model
# Elex 2030 - Testbed changes

## Description
Pulling out conformalization data from unit and agg cases for gaussian model.
## Jira Ticket

## Test Steps
Please check output of some examples to make sure correct conform data is coming out. And see if there's a less stupid way to do this.
